### ISSUE TITLE
Posizione circolari sulla home

### ISSUE THREAD

**Issue by Simsimus** [italia/design-scuole-wordpress-theme#645]
### Prerequisiti

- [X] Ho già [cercato](https://github.com/italia/design-scuole-wordpress-theme/issues?utf8=%E2%9C%93&q=is%3Aissue) tra le issue la presenza di un problema simile
- [X] Ho letto le [linee guida di contribuzione](https://github.com/italia/design-scuole-wordpress-theme/blob/main/CONTRIBUTING.md)

### Proposta

Grazie all'ottimo contributo di @sviluppoRobyone è possibile inserire un numero maggiore di articoli per tipologia e se visualizzarli in orizzontale o in verticale, ma la visualizzazione delle circolari rimane sempre come ultima posizione.
Capita di avere l'esigenza di avere in prima posizione le circolari e poi gli articoli; è per caso possibile introdurre nel pannello "configurazione" la possibilità di scegliere se le circolari devono essere visibili in prima posizione o dopo gli articoli?

### Motivazione e contesto

Rendere selezionabile la posizione delle circolari.

**Comment by sviluppoRobyone** [comment]
Se si vuole aggiungere funzionalità di riordino, sarebbe meglio includere anche gli eventi in questa logica così da prendere tutti e tre i tipi

**Comment by Simsimus** [comment]
Concordo, può essere utile anche scegliere dove posizionare gli eventi, ma le preferenze maggiormente segnalate sono "circolari in alto ed articoli subito dopo" oppure il contrario (in pratica com'è attualmente), ovvero "articoli in alto e circolari subito dopo"; gli eventi in genere vanno bene alla fine. 
@sviluppoRobyone pensi di riuscire ad apportare tale miglioria per ottimizzare l'ottimo lavoro che hai fatto?

**Comment by sviluppoRobyone** [comment]
La volontà c'è, rimane da capire quale sia la soluzione più efficace e di qualità.
Vedendo attualmente i tipi di campi CMB2 a disposizione, l'unica soluzione è quella di mettere un campo "Ordine di visualizzazione" con le combinazioni delle posizioni dei 3, cosa che non convince al momento...

**Comment by Simsimus** [comment]
Immagino sia una soluzione meno qualitativa, ma cosa più veloce potrebbe essere:
1) creare una opzione nella configurazione del tipo [circolari]: sopra articoli/sotto articoli;
2) inserire un IF ripetuto due volte (uno prima degli articoli ed uno dopo) nei files "novita-orizzontale.php" e "novita-verticale.php" che esegua l'attuale blocco di istruzioni a condizione che l'opzione corrisponda a quanto scelto dall'utente;
3) stessa logica dei primi due punti per gli "eventi" ma che per il momento eviterei per vedere come va con le circolari.

Lo so, è una cosa artigianale, ma credo sia risolutiva per il momento.
Tu che ne pensi?
